src/components/PhoneInput/countriesDropdown/styled/index.js

```diff
@@ -1,6 +1,6 @@
 import styled from 'styled-components';
 import attachThemeAttrs from '../../../../styles/helpers/attachThemeAttrs';
-import MenuArrowButton from '../../../Picklist/menuArrowButton';
+import CheckmarkIcon from '../../../Option/checkmark';
```

Comment:
I think we should not get this icon from other component folders since changes to Options could break PhoneInput, in this case it is better to repeat the icon to be inside PhoneInput folder, we are working in a solution in order to not repeat icons any more